swarm/network/stream/common_test.go

```diff
@@ -175,7 +113,7 @@ func newStreamerTester(t *testing.T) (*p2ptest.ProtocolTester, *Registry, *stora
 	db := storage.NewDBAPI(localStore)
 	delivery := NewDelivery(to, db)
 	streamer := NewRegistry(addr, delivery, db, state.NewInmemoryStore(), &RegistryOptions{
-		SkipCheck: defaultSkipCheck,
+		SkipCheck: false,
```

Comment:
false is the default value for bool, so SkipCheck can be removed, and by removing it, there is no need for RegistryOptions, and the last argument for NewRegistry here can be just nil.